packages/esm-covid-app/src/home.component.tsx

```diff
@@ -0,0 +1,16 @@
+import { OHRIWelcomeSection } from '@ohri/openmrs-esm-ohri-commons-lib';
+import React from 'react';
+import CovidHomePatientTabs from './views/dashboard/patient-list-tabs/covid-patient-list-tabs.component';
+import CovidSummaryTiles from './views/dashboard/summary-tiles/covid-summary-tiles.component';
+
+const Homecomponent = () => {
+  return (
+    <div style={{ margin: '1rem' }}>
+      <OHRIWelcomeSection title="COVID-19 Cases" />
+      <CovidSummaryTiles />
+      <CovidHomePatientTabs />
+    </div>
+  );
```

Comment:
I believe this where the designs vary

Reply:
Let me address that

Reply:
https://github.com/UCSF-IGHS/openmrs-esm-ohri/assets/19533785/b9a03681-bd76-4df7-82e9-c2fa37f30538

This is the updated video, thoughts @pirupius